examples/interrupt/uart/src/main.rs

```diff
@@ -0,0 +1,83 @@
+//! Example of responding to UART interrupts.
```

Comment:
You can probably extend this a bit, to send and to expect break conditions, too. That way, this example could be flashed on two devices and be used to showcase this functionality (instead of having an additional example that only generates breaks).

Reply:
Resolved in [4e95ac4](https://github.com/esp-rs/esp-hal/pull/4284/commits/4e95ac421177c346749a604e118996e5f07041bd)